wandb/sdk/internal/job_builder.py

```diff
@@ -154,10 +162,16 @@ def _build_repo_job(
 
         else:
             full_program_path = program_relpath
+
+        executable = metadata.get("python") or os.path.basename(sys.executable)
+        if "python" not in executable:
+            # if the executable comes from a manually created job, its just the version
```

Comment:
Wait so our executable for proto jobs is just the python version? Why is this necessary, couldn't we just add `python` to it?

Reply:
I don't see how this interacts with build, this is modifying how we set the entrypoint for the jobs we build. Build python version is set from the runtime key for jobs.

I think Im mostly confused about why we need to inject python in some cases now?

Reply:
@KyleGoyette  Can we huddle on this? I'm pretty sure it works, but very possibly not the optimal path.